crowdsourcing/models.py

```diff
@@ -1,13 +1,15 @@
 from django.contrib.auth.models import User
 from django.db import models
+from model_utils import Choices
```

Comment:
We can always achieve everything without dependencies. We decide whether we want to use a simpler structure without reinventing it.